yt_dlp/extractor/urplay.py

```diff
@@ -112,18 +112,19 @@ def parse_lang_code(code):
                 lang = ISO639Utils.short2long(lang)
             return lang or None
 
-        for k, v in (urplayer_data['streamingInfo'].get('sweComplete') or {}).items():
-            if (k in ('sd', 'hd') or not isinstance(v, dict)):
-                continue
-            lang, sttl_url = (v.get(kk) for kk in ('language', 'location', ))
-            if not sttl_url:
-                continue
-            lang = parse_lang_code(lang)
-            if not lang:
-                continue
-            sttl = subtitles.get(lang) or []
-            sttl.append({'ext': k, 'url': sttl_url, })
-            subtitles[lang] = sttl
+        for langpkg in urplayer_data['streamingInfo']:
+            for k, v in urplayer_data['streamingInfo'].get(langpkg).items():
```

Comment:
Why not:
```suggestion
        for langpkg in urplayer_data['streamingInfo'].values():
            for k, v in langpkg.items():
```

(Change var name. It's not clear what `langpkg` means)

Reply:
> (Change var name. It's not clear what langpkg means)

In case u missed.

Reply:
Thanks for the fix